src/styles/_supported-device-table.scss

```diff
@@ -92,6 +92,105 @@ $row-padding: 8px;
     }
   }
 
+  .seam-brand-section {
+    margin-bottom: 32px;
+    position: relative;
+
+    .seam-header {
+      display: flex;
+      align-items: center;
+      height: 64px;
+      cursor: pointer;
+
+      &:hover {
+        background: colors.$bg-aa;
+      }
+
+      .seam-brand-image {
+        width: 48px;
+        height: 48px;
+        padding: 0 8px;
+        box-sizing: content-box;
+        margin-right: 8px;
+      }
+
+      .seam-brand-name {
+        font-size: 21px;
+        line-height: 132%;
+        flex: 1;
+      }
+
+      .chevron {
+        transform: rotate(90deg);
+        margin-right: 24px;
+      }
+    }
+
+    .show-all-devices-button {
+      cursor: pointer;
+      background: #fff;
```

Comment:
Color var? Maybe we should lint against color literals outside of the var file.

Reply:
I usually don't bother with `#fff` / `#000000` because everybody knows what they are, but I've updated it in https://github.com/seamapi/react/pull/251/commits/5bc17746e93d7326ae2137c66ff02928b39b37b6 for consistency.